src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBox/TextBox_IsReadOnly.xaml.cs

```diff
@@ -0,0 +1,38 @@
+using Uno.UI.Samples.Controls;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.UI.Xaml;
+#if NETFX_CORE
```

Comment:
You're not suppose to need those pragma anymore.

Reply:
removed them